pandas/tests/series/indexing/test_multiindex.py

```diff
@@ -0,0 +1,42 @@
+""" test get/set & misc """
+
+from datetime import timedelta
+
+import numpy as np
+import pytest
+
+from pandas.core.dtypes.common import is_scalar
+
+import pandas as pd
+from pandas import (
+    Categorical,
+    DataFrame,
+    IndexSlice,
```

Comment:
you are going to fail linting here. see https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#python-pep8-black

for how to install pre-commit hooks to avoid this issue